style.css

```diff
@@ -0,0 +1,71 @@
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  display: flex;
+  margin: 20px;
+}
+
+.logo {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 25px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.onion {
+  margin-left: auto;
+}
+
+main {
+  background-image: url("images/header-shapes.svg");
+  background-size: 100%;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] Please, make sure to use the correct margins in your design to respect the original Figma design as much as possible. This is regarding the empty white lines that you have all around your main HTML file. Respecting the original design as much as possible will make you take care of details and that will make you a more valuable developer rather than any other that does not respect it.